package.json

```diff
@@ -27,7 +27,14 @@
     "require": [
       "babel-register",
       "babel-polyfill"
-    ]
+    ],
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-3",
+        "react"
+      ]
+    }
```

Comment:
Without it we're getting errors at the spread operator @ `/test/_transform.js: Unexpected token (10:8)`. I didn't immediately find a solution for it, it would be nice if you can tackle this problem :).